### DISCUSSION TITLE
deploy_gae.sh aborts with "...wait for operation..." error

### DISCUSSION THREAD

**Discussion by BalaKasi** [FIRST-Tech-Challenge/fmltc#231] [Q&A]
The last script deploy_gae.sh, aborts with below message - ERROR: (gcloud.app.deploy) Aborting wait for operation https://appengine.googleapis.com/v1/apps/<project-id>/operations/c8f14919-a378-4201-93c6-ca4c1672593f.

Also, when I tried logging with my team# and teamcode, I got this error - "The toolchain server is sad right now. Headquarters has been notified".

![image](https://user-images.githubusercontent.com/13141179/144330129-b4cc7b75-5ba8-42e1-8d46-70151b39dba3.png)

**Comment by lizlooney** [comment]
Did you try running deploy_gae.sh again?

**Comment by cmacfarl** [comment]
Did you upload a team info file per these instructions?

https://github.com/FIRST-Tech-Challenge/fmltc#create-and-upload-the-team_infoteams-file

**Reply by BalaKasi** [reply]
Got it - the file name is expected to be teams and not teams.txt!  So, now I get this error:
FailedPrecondition('no matching index found.')
![image](https://user-images.githubusercontent.com/13141179/144334853-828cb351-5c15-49a2-a15f-99a6afb60c7e.png)

**Comment by lizlooney** [comment]
Did you already run deploy_indexes.sh? Sometimes it take a little while for it to actually create all the indexes.

**Reply by BalaKasi** [reply]
Thanks!  I somehow overlooked that one script.  Looks like I am all set.
![image](https://user-images.githubusercontent.com/13141179/144337699-41f8fc16-17aa-4686-8092-cb77065a1acd.png)
